include/site/linux

```diff
@@ -66,9 +66,12 @@ ac_cv_sctp=no
 ac_cv_sys_restartable_syscalls=yes
 ac_cv_time_r_type=POSIX
 ac_cv_type_suseconds_t=yes
+ac_cv_size_t=yes
+ac_cv_ssize_t=yes
```

Comment:
Why are you adding these?

Reply:
Aren't they present in all platforms?

Reply:
yes I think so. If you want to add them please do it in a separate commit with a description in the commit message.